sklearnex/dispatcher.py

```diff
@@ -16,6 +16,7 @@
 
 import os
 import sys
+import warnings
```

Comment:
i guess we need to use unified mechanism with logging ?

Reply:
Would you prefer logging a warning to the `sklearnex` logger? We're not using logging anywhere so far in this part.

Reply:
I think it make sense to extend sklearnex logger for that. https://github.com/intel/scikit-learn-intelex/blob/86b392d5a62e214765dec324cc8f1453f4dba091/sklearnex/_utils.py#L49